Template/Variable/CustomRequestProcessingVariable.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Plugins\TagManager\Template\Variable;
+
+use Piwik\Piwik;
+use Piwik\Settings\FieldConfig;
+use Piwik\Validators\NotEmpty;
+
+class CustomRequestProcessingVariable extends BaseVariable
+{
+    const ID = 'CustomRequestProcessing';
+
+    public function getId()
+    {
+        return self::ID;
+    }
+
+    public function getCategory()
+    {
+        return self::CATEGORY_OTHERS;
+    }
+
+    public function isCustomTemplate()
+    {
+        return true;
+    }
+
+    public function getParameters()
+    {
+        return array(
+            $this->makeSetting('jsFunction', "function(request){\n\treturn request;\n}", FieldConfig::TYPE_STRING, function (FieldConfig $field) {
```

Comment:
@EPNW-Eric Do we need this `\n` here?

Reply:
Nope, that was just personal preference I guess...
But you are correct, the exisiting code in https://github.com/matomo-org/tag-manager/blob/e3c5a5ecd5b1e968f2bba98eaf1e19909ac78ebf/Template/Variable/CustomJsFunctionVariable.php#L36
 also does not use `\n`, so I will remove it here.